app/scripts/directives/about.js

```diff
@@ -21,25 +21,35 @@
       },
       link: function postLink(scope, element) {
         var main = element.closest('main'),
-          headerHeight = $('header').height();
+            headerHeight = $('header').height(),
+            html = $('html'),
+            aboutMenu = $('div.about');
 
         scope.menuVisible = false;
 
-        element.bind('click', function() {
-          console.log('click');
-          var aboutMenu = $('div.about');
-          aboutMenu.slideToggle();
-          scope.menuVisible = !scope.menuVisible;
-        });
-
         scope.toggleMenu = function(aboutMenu) {
           aboutMenu.slideToggle();
           scope.menuVisible = !scope.menuVisible;
         };
 
-        main.bind('scroll', function () {
+        element.bind('click', function(e) {
+          e.stopPropagation();
+          var aboutMenu = $('div.about');
```

Comment:
Couldn't these aboutMenu selections be eliminated because you have the selection on line 26.